### ISSUE TITLE
HELP: Editing Config File / NFS share typo

### ISSUE THREAD

**Issue by holocronology** [HclX/WyzeHacks#89]
I'm testing this on a V2 and starting with firmware 4.9.5.115, so I'm doing the SD card install. Initially it didn't seem to work, but when I checked this morning telnet access was working. Maybe it just took some time.

However, I'm having some issue with NFS. I have looked over the notes, but maybe my telneting is rusty (it is, not maybe) and I'm not figuring out how to update it. Specifically, I noticed an typo in my initial config.int file. So my question is how can I edit the NFS share location?

A follow-up: I'm running a Netgear router with ReadyShare and I have the network share location active. From windows I would normally use \\ROUTER_NAME\SHARE_NAME to access this. For the NFS would I use:

1. \\ROUTER_NAME\SHARE_NAME
2. ROUTER_IP:/SHARE_NAME
3. ROUTERNAME:/SHARE_NAME

I could test these on my own if I could figure out how to change it via telnet.

**Comment by holocronology** [comment]
Okay, I withdraw the follow-up part of this, as I did some digging in the closed issues and figured out that my Netgear router is doing an SMB share. So, looks like I just found a use for this Pi Zero I have sitting around doing nothing....time to set up an NFS share server.

**Comment by holocronology** [comment]
Just to follow up on this, I've tried all possible config options that I can figure out and it will not connect to the NFS share/server on the Pi. I tried setting up an NFS share on a windows server (I know, this is error prone) and it wouldn't work.

I can telnet in, but that's about it. Cool project, but the last of community engagement to help out a fellow adopter and the seeming preference to not include some other file connection options leads me to abandon this as even a side fun project.
